conf/supervisord.conf

```diff
@@ -25,3 +27,19 @@ events=PROCESS_LOG
 stderr_logfile=/dev/fd/1
 stderr_logfile_maxbytes=0
 
+[program:refresh_blocklist]
+command=sh -c "while true; do bin/fetch-blocklist; sleep 60; done"
+stdout_logfile=NONE
+stderr_logfile=NONE
+stdout_events_enabled=true
+stderr_events_enabled=true
+
+[unix_http_server]
+file = .supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl = unix://.supervisor.sock
+prompt = viahtml
```

Comment:
These two different files were very different, and I don't know why. I've made them much more similar

Reply:
Do we want `supervisor-ctl- config in prod though?

Reply:
I don't know, I don't think we necessarily want it in dev either?